components/TopNav/TopNav.tsx

```diff
@@ -14,7 +14,7 @@ import {
 import { useDocVersionContext } from "../../context/DocVersionContext";
 import { useNavBarCollapsedContext } from "../../context/NavBarCollapseContext";
 import { SearchContextProvider } from "../../context/SearchContext";
-import { ReactComponent as API } from "../../images/icons/api.svg";
+import { ReactComponent as Api } from "../../images/icons/api.svg";
```

Comment:
Let's use meaningful names for icons.
```suggestion
import { ReactComponent as ApiIcon } from "../../images/icons/api.svg";
```